src/components/site/exportAnswers/index.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import xlsx from 'json-as-xlsx';
+import { useTranslation, Trans } from 'gatsby-plugin-react-i18next';
+import { getContent } from '../../../lib/answers';
+import Button from '../button';
+
+const ExportAnswers = ({ name, pages, answers }) => {
+  const { t } = useTranslation();
+
+  const getTable = table => {
+    const { rows, rowTitles, colTitles } = table;
+
+    const flattenedTable = rows.reduce((currentRow, cells, rowIndex) => {
+      const flattenedRow = cells.reduce(
+        (flattenedCells, currentCell, cellIndex) => {
+          const { id } = currentCell;
+          const { data } = answers[id];
+
+          const cell = `${t(colTitles[cellIndex + 1])}: ${
+            Array.isArray(data) ? data.join('-') : data
+          } `;
+
+          return flattenedCells + cell;
+        },
+        `${t(rowTitles[rowIndex])} `
+      );
+
+      return currentRow + flattenedRow;
+    }, '');
+
+    return flattenedTable;
+  };
+
+  const getAnswer = question => {
+    const { id, answerAccessor, answerPre, answerPost } = question;
+    const { data } = answers[id];
+
+    let content = getContent(answerAccessor, data, true);
+
+    if (answerPre) {
+      content = t(answerPre) + content;
+    }
+
+    if (answerPost) {
+      content += t(answerPost);
+    }
+
+    if (typeof content === 'string') {
+      content = content.replace(/(<([^>]+)>)/gi, '');
+    }
+
+    return content;
+  };
+
+  const getContentRow = questions => {
+    const content = {
+      name,
+      answers: {},
+    };
+
+    questions.forEach(parentQuestion => {
+      const { number, question, tables } = parentQuestion;
+      if (tables && tables.length > 0) {
+        content.answers[number] = getTable(tables[0]);
```

Comment:
if there is a table associated with the parent question, send it to a special parser